src/TraceEvent/TraceEvent.csproj

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
+    <LangVersion>7.0</LangVersion>
```

Comment:
Including @sharwell 

Please remove this and update the <langVrsion> declaration in src\Directory.Build.props instead.   That way all projects benefit from it uniformly.  

I am OK moving to C# 7.  It means a dependency on VS 2017, but realistically we have probably already many dependencies on 2017  (msbuild files, nuget support, bug fixes, ...)

Reply:
@vancem done